reference/orders.v3.yml

```
@@ -2268,6 +2268,14 @@ components:
         - order_id
         - items
       x-internal: false
+      x-examples:
+        example-1:
+          order_id: 1
```

```suggestion
          order_id: 1
```
use an order_id that is more like what they'll encounter.